test/lib/index.ts

```diff
@@ -146,3 +146,15 @@ test('Performance: Parse big npm package-lock.json with cyclic deps and dev-deps
   );
   t.deepEqual(depTree.name, 'trucolor', 'Tree is created correctly');
 });
+
+test('Parse yarn yarn.lock', async (t) => {
```

Comment:
I would like to see a test where we handle invalid files to parse as JSON for both of the lockfile types.

Reply:
Tests added